istio.deps

```diff
@@ -4,13 +4,13 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "3a04f855cd176213f316b5f12d67c2a79e2ffdb9"
+    "lastStableSHA": "e75aa441f21df5f13e26f8517848616f10ebcec4"
```

Comment:
Can we use the latest sha (`5789f684ac335043e42b4a0242b42d4f48c3a0cd`)?

Reply:
Done.